mantidproject/mantid
# Allow packaging on macOS when ENABLE_MANTIDPLOT=OFF

**Description of work.**

Ensures that if MantidPlot is disabled in cmake with `-DENABLE_MANTIDPLOT` then the macOS package only contains the MantidWorkbench bundle and not other stray directories.



**To test:**

* First build with cmake flags `-DENABLE_CPACK=ON -DENABLE_MANTIDPLOT=ON -DENABLE_WORKBENCH=ON` and run the `package` target. The package should still contain both MantidPlot & Workbench.
* Now try disabling MantidPlot and check the package on contains workbench.
* A package without either is not yet supported nor is a package with only MantidPlot.

Fixes #28833
Refs #28650

*This does not require release notes* because **it is an internal issue.**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
